test/ui/test_reflectivity_extraction_global.py

```diff
@@ -109,9 +109,14 @@ def test_reflectivity_recalculated_on_config_change(mocker, qtbot):
     main_window.actionAddPlot.triggered.emit()
     assert mock_calculate_reflectivity.call_count == 2
 
+    # add a data tab - it is automatically initialized with the two runs from the first data tab
+    main_window.addDataTable()
+    nbr_data_tabs = main_window.data_tab_count
```

Comment:
can you change to `number_data_tabs`? It took me some time to figure out what nbr stands for